src/main/java/pl/lodz/p/it/zzpj/botsite/web/controllers/UserTaskAdminController.java

```diff
@@ -0,0 +1,103 @@
+package pl.lodz.p.it.zzpj.botsite.web.controllers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.*;
+import pl.lodz.p.it.zzpj.botsite.entities.UserTask;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.deletion.UserTaskDeletionException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.notfound.UserNotFoundException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.notfound.UserTaskNotFoundException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.retrieval.UserTaskRetrievalException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.saving.UserTaskAdditionException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.saving.UserTaskUpdateException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.unconsistent.UserTaskIdAlreadyExistsException;
+import pl.lodz.p.it.zzpj.botsite.services.UserTaskService;
+import pl.lodz.p.it.zzpj.botsite.web.dto.usertasks.UserTaskAdminDTO;
+import pl.lodz.p.it.zzpj.botsite.web.dto.usertasks.UserTaskUserDTO;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@Slf4j
+@RequestMapping("/api/usertaskAdmin")
+public class UserTaskAdminController {
+
+    private final ModelMapper modelMapper;
+    private final UserTaskService userTaskService;
+
+    @Autowired
+    public UserTaskAdminController(ModelMapper modelMapper,
+                              UserTaskService userTaskService) {
+        this.modelMapper = modelMapper;
+        this.userTaskService = userTaskService;
+    }
+
+    //GET ALL FOR USER
+    @Secured("ROLE_ADMIN")
+    @GetMapping(
+            value = "user/{userId}",
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public List<UserTaskAdminDTO> getAllByUserId(@PathVariable("userId") final Long userId)
+            throws UserTaskNotFoundException, UserNotFoundException, UserTaskUpdateException {
+        List<UserTask> userTaskList = userTaskService.getListOfUserTasksByUserId(userId);
+        java.lang.reflect.Type targetListType = new TypeToken<List<UserTaskAdminDTO>>() {}.getType();
+        List<UserTaskAdminDTO> userTaskAdminDTOS = modelMapper.map(userTaskList, targetListType);
+        return userTaskAdminDTOS;
+    }
+
+    // GET SINGLE TASK
+    @Secured("ROLE_ADMIN")
+    @GetMapping(
+            value = "/{id}",
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    @ResponseStatus(HttpStatus.OK)
+    public UserTaskAdminDTO getTaskByUserId(@PathVariable("id") final Long taskId)
+            throws UserTaskRetrievalException {
+        UserTask userTask = userTaskService.findById(taskId);
+        return modelMapper.map(userTask, UserTaskAdminDTO.class);
```

Comment:
Nie podoba mi się jak używacie wszędzie nazw bez **this.**. Wiem, że język na to pozwala, ale **this.** sugeruje jasno, że chodzi o pole, natomiast bez wyspecyfikowania tego trzeba się zastanawiać, a to już źle.